build/ts/esm-js.js

```diff
@@ -0,0 +1,15 @@
+import { glob } from "glob";
+import fs from "fs";
+import path from 'path';
+
+const files = await glob("./protos/node/**/*.ts")
+
+//
+// Add .js to imports so they are esm compliant
+files.forEach(file => {
```

Comment:
We add js to the generated file in place...and then let tsc compile to `node/protos`